src/test/header-manager.test.ts

```
@@ -0,0 +1,370 @@
+describe("Fonctions globales", () => {
+  beforeEach(() => {
+    vi.clearAllMocks();
+    
+    // Reset complètement les mocks des constructeurs
+    MockedScrollHandler.mockReset();
+    MockedMobileMenu.mockReset();
+    MockedLanguageManager.mockReset();
+    MockedThemeManager.mockReset();
+
+    // Reconfigurer les mocks
+    MockedScrollHandler.mockImplementation(() => ({ destroy: vi.fn() } as any));
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)